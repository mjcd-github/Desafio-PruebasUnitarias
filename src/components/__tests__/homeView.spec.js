import { shallowMount } from "@vue/test-utils";
import { describe, it, expect } from "vitest";
import HomeView from "@/views/HomeView.vue";

describe("HomeView", () => {
    it("Carga correctamente", () => {
        const wrapper = shallowMount(HomeView);
        expect(wrapper.html()).toMatchSnapshot();
    });
})