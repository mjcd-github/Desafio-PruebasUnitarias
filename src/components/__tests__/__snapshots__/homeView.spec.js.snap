// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeView > Carga correctamente 1`] = `
"<main>
  <the-welcome-stub></the-welcome-stub>
</main>"
`;
